src/draw_buffer.rs

```diff
@@ -113,8 +116,9 @@ impl DrawBuffer {
 
         try!(draw_buffer.init(context, color_attachment_type));
 
-        debug_assert!(gl::check_frame_buffer_status(gl::FRAMEBUFFER) == gl::FRAMEBUFFER_COMPLETE);
-        debug_assert!(gl::get_error() == gl::NO_ERROR);
+        debug_assert!(
```

Comment:
`debug_assert_eq!` would be much better here since it would print the actual error/status